### PR TITLE
CF template & Readme improvement

### PR DESCRIPTION
**Pull request by shimomut**
*Description of changes:*

- Create SETUP_COMPLETED.txt file when setting up finishes
- Explain how to wait the completion of setup
- Explain how to run the first lab
- Deleted unnecessary steps in how to clean up

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,19 +10,22 @@
...
 1. Visit [SageMaker Notebooks instances page](https://console.aws.amazon.com/sagemaker/home#/notebook-instances) and find "PanoramaWorkshop". Click "Open JupyterLab".
-1. From the file browser pane in left hand side, find "aws-panorama-immersion-day" directory, and double click it to see the contents of this repository.
-1. Open "setup-sm.ipynb". This notebook contains required environment setting steps.
-1. From menu bar, select "Run" > "Run All Cells"
+1. When you see the dialog ("Loading... The loading screen is taking a long time..."), please click "Keep Waiting" button, and wait until JupyterLab screen opens.
 
+    ![](images/loading_taking_time_dialog.png)
 
-## How to clean up
+1. Setup is continuing in the background. Please for the `"SETUP_COMPLETED.txt"` file to appear in the top level directory.
```
Comment (hagencoder) [inline README.md]: Should be "Please wait for the..."

**Review by hagencoder — COMMENTED** [review]
I added one comment in the readme that you should resolve. Other than that it looks good.

File: README.md (hunk 1 above)
Comment (shimomut) [reply README.md]: Thank you. Fixed.

**Review by hagencoder — APPROVED** [review]
Looks good now.

**Pull request merged**